person_capture/gui_app.py

```
@@ -5485,12 +6018,59 @@ def _save_hdr_crop_p010(
                 "-color_primaries", "bt2020",
                 "-color_trc", "smpte2084",
             ]
-        cmd.append(out_path)
+        out_ext = Path(out_path).suffix or ".mkv"
+        tmp_out = out_path + f".tmp{out_ext}"
+        cmd.append(tmp_out)
 
+        timeout_sec = max(5, int(getattr(self.cfg, "hdr_export_timeout_sec", 300) or 300))
         try:
-            subprocess.run(cmd, check=True)
+            cp = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)